backend/src/akvo/lumen/endpoint/aggregation.clj

```diff
@@ -23,7 +24,16 @@
                              visualisation-type
                              (json/parse-string query keyword))
           (catch JsonParseException e
-            (http/bad-request {:message (.getMessage e)})))
+            (log/error e)
+            (http/bad-request
+             {:message (format "JSON parse exception: %s" (.getMessage e))}))
+          (catch java.sql.SQLException e
```

Comment:
We can `(import java.sql.SQLException)` in the namespace declaration, and use `SQLException`